src/styles/blocks/header.scss

```diff
@@ -0,0 +1,56 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0.1px 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    padding: 10px 50px;
+  }
+
+  &__nav-list {
+    display: flex;
+    list-style: none;
+    margin-right: 50px;
+  }
+  
+  &__nav-item {
+    padding: 0 10px;
+  }
+  
+  &__nav-link {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    align-items: center;
+    height: 60px;
+    color: black;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+  
+  &__nav-link:hover {
+    color: #00acdc;
+  }
+
+}
+
+.is-active {
+  color: #00acdc;
```

Comment:
As this color is used several times, I suggest making a variable of it, so that if the teamleader decides he doesn't like blue anymore, it would not lead you into making a lot of small changes)

Reply:
I see you have a variable for this color. It i used for the button. Why not renaming and pasting it here)

Reply:
> I see you have a variable for this color. It i used for the button. Why not renaming and pasting it here)

Haven't noticed that. Thanks for pointing out on my mistake:)